### DISCUSSION TITLE
CRPSS data are missing from output ECNT linetype records

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1725] [Configuration] [answered]
I am testing EnsembleStat verification for 30-member GEFS with single climo (just climo mean as reference) in METplus  The running is  successful. But checking the ensemble_stat output stat file I found the CRPSS data (#28 or #4  after ECNT) is missing (labeled as NA).
But if running this case with exactly same input files in purely MET (outside of METplus), the CRPSS data can be seen in ECNT linetype in the output ensemble_stat stat file.  
So I guess there is some problem in my METplus conf file, but can not figure out where is the problem.
If needed, I can copy entire sample onto HERA. 
The final conf file is printed below

[config]
LOOP_ORDER = times
PROCESS_LIST = EnsembleStat
OMP_NUM_THREADS = 1
GEMPPAKTOCF_JAR = GempakToCF.jar
LOOP_BY = VALID
VALID_TIME_FMT = %Y%m%d%H
VALID_BEG = {ENV[vday]}{ENV[vbeg]}
VALID_END = {ENV[vday]}{ENV[vend]}
METPLUS_PATH = {ENV[METPLUS_PATH]}
VALID_INCREMENT = {ENV[valid_increment]}
ENSEMBLE_STAT_ENS_VLD_THRESH = 0.5
ENSEMBLE_STAT_ENS_THRESH = 0.5
ENS_VAR1_NAME = HGT
ENS_VAR1_LEVELS = P500
BOTH_VAR1_NAME = {ENS_VAR1_NAME}
BOTH_VAR1_LEVELS = {ENS_VAR1_LEVELS}
lead = {ENV[lead]}
LEAD_SEQ = {lead}
MODEL = {ENV[MODEL]}
model = {ENV[model]}
ENSEMBLE_STAT_REGRID_TO_GRID = {ENV[regrid]}
OBTYPE = {ENV[OBTYPE]}
MASKPATH = {ENV[maskpath]}
ENSEMBLE_STAT_DESC = NA
ENSEMBLE_STAT_REGRID_METHOD = NEAREST
ENSEMBLE_STAT_REGRID_WIDTH = 1
ENSEMBLE_STAT_REGRID_VLD_THRESH = 0.5
ENSEMBLE_STAT_REGRID_SHAPE = SQUARE
METPLUS_ENS_SSVAR_BIN_SIZE = 1000000
METPLUS_ENS_PHIST_BIN_SIZE = 0.05
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_METHOD = BILIN
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_WIDTH = 2
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_VLD_THRESH = 0.5
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_SHAPE = SQUARE
ENSEMBLE_STAT_CLIMO_MEAN_FIELD =
ENSEMBLE_STAT_CLIMO_MEAN_TIME_INTERP_METHOD = DW_MEAN
ENSEMBLE_STAT_CLIMO_MEAN_MATCH_MONTH = FALSE
ENSEMBLE_STAT_CLIMO_MEAN_DAY_INTERVAL = 31
ENSEMBLE_STAT_CLIMO_MEAN_HOUR_INTERVAL = 6
ENSEMBLE_STAT_MASK_GRID = G003
ENSEMBLE_STAT_MASK_POLY =
METPLUS_CI_ALPHA = 0.05
ENSEMBLE_STAT_NBRHD_PROB_WIDTH = 3
ENSEMBLE_STAT_NBRHD_PROB_SHAPE = CIRCLE
ENSEMBLE_STAT_NBRHD_PROB_VLD_THRESH = 0.0
ENSEMBLE_STAT_NMEP_SMOOTH_VLD_THRESH = 0.0
ENSEMBLE_STAT_NMEP_SMOOTH_SHAPE = CIRCLE
ENSEMBLE_STAT_NMEP_SMOOTH_GAUSSIAN_DX = 81.27
ENSEMBLE_STAT_NMEP_SMOOTH_GAUSSIAN_RADIUS = 120
ENSEMBLE_STAT_NMEP_SMOOTH_METHOD = GAUSSIAN
ENSEMBLE_STAT_NMEP_SMOOTH_WIDTH = 1
ENSEMBLE_STAT_INTERP_FIELD = BOTH
ENSEMBLE_STAT_INTERP_VLD_THRESH = 1.0
ENSEMBLE_STAT_INTERP_SHAPE = SQUARE
ENSEMBLE_STAT_INTERP_TYPE_METHOD = NEAREST
ENSEMBLE_STAT_INTERP_TYPE_WIDTH = 1
ENSEMBLE_STAT_OUTPUT_FLAG_ECNT = BOTH
ENSEMBLE_STAT_OUTPUT_FLAG_RPS = NONE
ENSEMBLE_STAT_OUTPUT_FLAG_RHIST = NONE
ENSEMBLE_STAT_OUTPUT_FLAG_PHIST = NONE
ENSEMBLE_STAT_OUTPUT_FLAG_ORANK = NONE
ENSEMBLE_STAT_OUTPUT_FLAG_SSVAR = NONE
ENSEMBLE_STAT_OUTPUT_FLAG_RELP = NONE
ENSEMBLE_STAT_ENSEMBLE_FLAG_LATLON = TRUE
ENSEMBLE_STAT_ENSEMBLE_FLAG_MEAN = TRUE
ENSEMBLE_STAT_ENSEMBLE_FLAG_STDEV = TRUE
ENSEMBLE_STAT_ENSEMBLE_FLAG_MINUS = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_PLUS = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_MIN = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_MAX = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_RANGE = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_VLD_COUNT = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_FREQUENCY = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_NEP = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_NMEP = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_RANK = FALSE
ENSEMBLE_STAT_ENSEMBLE_FLAG_WEIGHT = FALSE
GRID_WEIGHT_FLAG = COS_LAT
ENSEMBLE_STAT_MET_CONFIG_OVERRIDES = tmp_dir = "{TMP_DIR}"; climo_mean = {field = [{ name="HGT"; level="P500"; } ];};
modelhead = {ENV[modelhead]}
modelpath = {ENV[modelpath]}
modelgrid = {ENV[modelgrid]}
modeltail = {ENV[modeltail]}
gdaspath = {ENV[gdaspath]}
gdashead = {ENV[gdashead]}
gdasgrid = {ENV[gdasgrid]}
climhead = {ENV[climhead]}
climpath = {ENV[climpath]}
climgrid = {ENV[climgrid]}
climtail = {ENV[climtail]}
members = {ENV[members]}
ENSEMBLE_STAT_CONFIG_FILE = {METPLUS_BASE}/parm/met_config/EnsembleStatConfig_wrapped
FCST_ENSEMBLE_STAT_INPUT_DATATYPE = GRIB
OBS_ENSEMBLE_STAT_INPUT_GRID_DATATYPE = GRIB
ENSEMBLE_STAT_N_MEMBERS = {members}
ENSEMBLE_STAT_OUTPUT_PREFIX = {MODEL}_g2g_BIN1_FHR{lead?fmt=%3H}
FCST_ENSEMBLE_STAT_INPUT_DIR = {modelpath}
OBS_ENSEMBLE_STAT_GRID_INPUT_DIR = {gdaspath}
ENSEMBLE_STAT_OUTPUT_DIR = {OUTPUT_BASE}/stat/{model}
ENSEMBLE_STAT_CLIMO_MEAN_INPUT_DIR = {climpath}
ENSEMBLE_STAT_CLIMO_STDEV_INPUT_DIR = {climpath}
FCST_ENSEMBLE_STAT_INPUT_TEMPLATE = atmos.{init?fmt=%Y%m%d}/{modelhead}/{modelhead}.ens??.t{init?fmt=%H}z.{modelgrid}{lead?fmt=%3H}{modeltail}
OBS_ENSEMBLE_STAT_GRID_INPUT_TEMPLATE = atmos.{valid?fmt=%Y%m%d}/{modelhead}/{gdashead}.t{valid?fmt=%H}z.{gdasgrid}.f000.grib2
ENSEMBLE_STAT_CLIMO_MEAN_INPUT_TEMPLATE = mean_{valid?fmt=%m%d}
ENSEMBLE_STAT_CLIMO_STDEV_INPUT_TEMPLATE = stdev_{valid?fmt=%m%d}
ENSEMBLE_STAT_SKIP_TIMES =
SKIP_TIMES =
ENSEMBLE_STAT_MANDATORY = True
ENSEMBLE_STAT_SKIP_IF_OUTPUT_EXISTS = False
FCST_IS_PROB = False
OBS_IS_PROB = False
ENS_ENSEMBLE_STAT_INPUT_DATATYPE =
OBS_ENSEMBLE_STAT_INPUT_POINT_DATATYPE =
OBS_ENSEMBLE_STAT_POINT_INPUT_DIR =
ENSEMBLE_STAT_CTRL_INPUT_DIR =
ENSEMBLE_STAT_CLIMO_MEAN_FILE_NAME = /lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix/climos/atmos/era_interim/mean_{valid?fmt=%m%d}
ENSEMBLE_STAT_CLIMO_STDEV_FILE_NAME = /lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix/climos/atmos/era_interim/stdev_{valid?fmt=%m%d}
ENSEMBLE_STAT_CLIMO_STDEV_USE_FCST = False
ENSEMBLE_STAT_CLIMO_STDEV_USE_OBS = False
ENSEMBLE_STAT_MET_OBS_ERR_TABLE =
INIT_SEQ =

[user_env_vars]

[runtime]
CLOCK_TIME = 20220802150437
METPLUS_VERSION = 4.1.1
MET_INSTALL_DIR = {ENV[MET_ROOT]}
CONFIG_INPUT = /apps/ops/para/libs/intel/19.1.3.304/metplus/4.1.1/parm/metplus_config/metplus_system.conf,/apps/ops/para/libs/intel/19.1.3.304/metplus/4.1.1/parm/metplus_config/defaults.conf,/lfs/h2/emc/vpppg/noscrub/binbin.zhou/EVS/evs.v1.0/parm/metplus_config/machine.conf,/lfs/h2/emc/vpppg/noscrub/binbin.zhou/EVS/evs.v1.0/parm/metplus_config/global_ens/atmos_grid2grid/stats/EnsembleStat_fcstGEFS_obsModelAnalysis_climoERAI.conf
METPLUS_CONF = {OUTPUT_BASE}/final_ens_{model}_{ENV[vbeg]}_upper.conf
TMP_DIR = {OUTPUT_BASE}/tmpSTAGING_DIR = {OUTPUT_BASE}/stageCONVERT = convert
GEMPAKTOCF_JAR = GempakToCF.jar
GFDL_TRACKER_EXEC = /path/to/standalone_gfdl-vortextracker_v3.9a/trk_exec
INPUT_MUST_EXIST = True
USER_SHELL = bash
DO_NOT_RUN_EXE = False
SCRUB_STAGING_DIR = False
MET_BIN_DIR = {MET_INSTALL_DIR}/{ENV[MET_bin_exec]}
INPUT_BASE = /path/to
OUTPUT_BASE = {ENV[output_base]}
LOG_METPLUS = /lfs/h2/emc/ptmp/binbin.zhou/evs/tmpnwprd/evs_gefs_grid2grid_stats.253993/grid2grid/run_gefs_valid_at_t00z_fhr1/logs/metplus.log.20220802150437
LOG_DIR = {OUTPUT_BASE}/logs
LOG_TIMESTAMP_TEMPLATE = %Y%m%d%H%M%S
LOG_TIMESTAMP_USE_DATATIME = no
LOG_MET_OUTPUT_TO_METPLUS = {ENV[log_met_output_to_metplus]}
LOG_LEVEL = DEBUG
LOG_MET_VERBOSITY = {ENV[met_verbosity]}
LOG_LINE_FORMAT =
LOG_ERR_LINE_FORMAT = {LOG_LINE_FORMAT}
LOG_DEBUG_LINE_FORMAT = {LOG_LINE_FORMAT}
LOG_INFO_LINE_FORMAT = %(asctime)s.%(msecs)03d %(name)s %(levelname)s: %(message)s
LOG_LINE_DATE_FORMAT = %m/%d %H:%M:%S
METPLUS_BASE = /apps/ops/para/libs/intel/19.1.3.304/metplus/4.1.1
PARM_BASE = /apps/ops/para/libs/intel/19.1.3.304/metplus/4.1.1/parm
LOG_ENSEMBLE_STAT_VERBOSITY = 2
LOG_TIMESTAMP = 20220802150437

**Comment by BinbinZhou-NOAA** [comment]
More info for this case: 
If removing the climo  files from the conf file, including command out  all *CLIMO* parameters, the output has exactly same results. This implies the reference climo data files do not take effect in the EnsembleStat verification. The ECENT line type data are shown below:
V10.1.1 GEFS  NA   240000    20220725_000000 20220725_000000 000000   20220725_000000 20220725_000000 HGT      gpm        P500     HGT     gpm       P500    GDAS   G003    NEAREST     1           NA          NA         NA         NA    ECNT      65160  30  3.22199 NA   3.08695   -0.43917   6.19634  9.48357  NA  NA  NA  NA  NA  3.27639  NA  NA

**Comment by JohnHalleyGotway** [comment]
@BinbinZhou-NOAA I see that you're getting differing results when running Ensemble-Stat directly on the command line versus running it via METplus. I agree with your diagnosis. The likely culprit is in how the configuration is being set via METplus.

I want to ask about these 2 settings from what you provided above:
```
ENSEMBLE_STAT_CLIMO_MEAN_FIELD =
...
ENSEMBLE_STAT_MET_CONFIG_OVERRIDES = tmp_dir = "{TMP_DIR}"; climo_mean = {field = [{ name="HGT"; level="P500"; } ];};
```

It would seem that the climo mean input field is EMPTY... but then you're trying to define it via the OVERRIDES option. I do recall having a conversation with @georgemccabe about the handling of OVERRIDES. He'd be the best person to advise here.

But he's out of the office this week.

If you have time before the METplus telecon on Monday, please try moving the climo_mean setting out of ENSEMBLE_STAT_MET_CONFIG_OVERRIDES. Instead, specify it using the ENSEMBLE_STAT_CLIMO_MEAN_FIELD setting.

Does that produce any different results?

**Comment by BinbinZhou-NOAA** [comment]
John,

  Our dev machine was just down, so have to wait for its back. 
Sure I'll test it before the Monday meeting. Hope can find something.

Thanks!

Binbin

**Comment by BinbinZhou-NOAA** [comment]
I remember I have tested  your suggestion before. That is,  removed the  ENSEMBLE_STAT_MET_CONFIG_OVERRIDES statement
and set 
ENSEMBLE_STAT_CLIMO_MEAN_FIELD = { name="HGT"; level="P500"; } 
But seems same issue

My setting ENSEMBLE_STAT_CLIMO_MEAN_FIELD  is correct?

**Comment by BinbinZhou-NOAA** [comment]
I re-ran the case in which ENSEMBLE_STAT_MET_CONFIG_OVERRIDES is removed and set it in ENSEMBLE_STAT_CLIMO_MEAN_FIELD,
the results have no difference. CRPSS is still NA

**Comment by JohnHalleyGotway** [comment]
OK, let's touch base with @georgemccabe at the NOAA METplus telecon on Monday to help debug this behavior. Thanks.

**Comment by georgemccabe** [comment]
@BinbinZhou-NOAA could you send us the log file from this run with `LOG_LEVEL=DEBUG`?

**Reply by georgemccabe** [reply]
The MET config file used when calling ensemble_stat outside of METplus would also be useful to compare.

**Comment by BinbinZhou-NOAA** [comment]
I just sent both METplus and MET running samples to HERA in directory:
/scratch2/NCEPDEV/fv3-cam/noscrub/Binbin.Zhou 
2 directories for
METPlus sample:  evs_gefs_METplus_stats_crpss
   
MET sample:  evs_gefs_MET_stats_crpss

In evs_gefs_METplus_stats_crpss, 
  EnsembleStat_fcstGEFS_obsModelAnalysis_climoERAI.conf is original conf file
  run_gefs_valid_at_t00z_fhr1_g2g.sh defines all required environment variables
 sub-directory /grid2grid/run_gefs_valid_at_t00z_fhr1  is running output_base directory, in which
/logs contains log files
/data contains all input files (30 member files, analysis data, c;limo data files, etc)

in evs_gefs_MET_stats_crpss 
verf_g2g_ens_stat_config_bin1 is MET config file, and run.sh is execution script file
all data in /data sub-directory

**Comment by BinbinZhou-NOAA** [comment]
I just sent both METplus and MET running samples to HERA in directory:
/scratch2/NCEPDEV/fv3-cam/noscrub/Binbin.Zhou
2 directories for
METPlus sample: evs_gefs_METplus_stats_crpss

MET sample: evs_gefs_MET_stats_crpss

In evs_gefs_METplus_stats_crpss,
EnsembleStat_fcstGEFS_obsModelAnalysis_climoERAI.conf is original conf file
run_gefs_valid_at_t00z_fhr1_g2g.sh defines all required environment
variables
sub-directory /grid2grid/run_gefs_valid_at_t00z_fhr1 is running output_base
directory, in which
/logs contains log files
/data contains all input files (30 member files, analysis data, c;limo data
files, etc)

in evs_gefs_MET_stats_crpss
verf_g2g_ens_stat_config_bin1 is MET config file, and run.sh is execution
script file
all data in /data sub-directory

On Mon, Aug 8, 2022 at 11:30 AM George McCabe ***@***.***>
wrote:

> The MET config file used when calling ensemble_stat outside of METplus
> would also be useful to compare.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/1725#discussioncomment-3350602>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOPKM4KJG3E44RBDJLVYERZDANCNFSM55MA7ZQA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

IMSG at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

**Comment by georgemccabe** [comment, marked as answer]
Hi @BinbinZhou-NOAA, thank you for making these files available.

(1) I noticed in your MET config file that you have the following set:

`climo_mean = fcst;`

This is typically used to get the field information from the fcst dictionary and use it for the climo_mean field info. There is a METplus setting to accomplish this called [ENSEMBLE_STAT_CLIMO_MEAN_USE_FCST ](https://metplus.readthedocs.io/en/latest/Users_Guide/glossary.html#term-ENSEMBLE_STAT_CLIMO_MEAN_USE_FCST). Set this variable to **True** instead of setting climo_mean.field explicitly.

`ENSEMBLE_STAT_CLIMO_MEAN_USE_FCST = True`

(2) In your METplus log file, I'm noticing that the settings for climo_mean are using some values that differ from the MET config file settings:

> 08/02 15:04:37.448 metplus INFO: METPLUS_CLIMO_MEAN_DICT=climo_mean = {file_name = ["/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix/climos/atmos/era_interim/mean_0725"];regrid = {method = BILIN;width = 2;vld_thresh = 0.5;shape = SQUARE;}time_interp_method = DW_MEAN;match_month = FALSE;day_interval = 31;hour_interval = 6;}

I see that the climo_mean.regrid dictionary is set to values that differ from the defaults via METplus. These are unset in the MET config, so the default values are used.

Looking at your METplus config file (evs_gefs_METplus_stats_crpss/EnsembleStat_fcstGEFS_obsModelAnalysis_climoERAI.conf), I notice that these settings do not match what is being set in the METPLUS_CLIMO_MEAN_DICT environment variable:

```
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_METHOD =
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_WIDTH =
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_VLD_THRESH =
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_SHAPE =
ENSEMBLE_STAT_CLIMO_MEAN_FIELD =
#ENSEMBLE_STAT_CLIMO_MEAN_FIELD = { name="HGT"; level="P500"; }
ENSEMBLE_STAT_CLIMO_MEAN_TIME_INTERP_METHOD = DW_MEAN
ENSEMBLE_STAT_CLIMO_MEAN_MATCH_MONTH = TRUE
ENSEMBLE_STAT_CLIMO_MEAN_DAY_INTERVAL =
ENSEMBLE_STAT_CLIMO_MEAN_HOUR_INTERVAL =
ENSEMBLE_STAT_CLIMO_MEAN_TIME_STEP = 21600
```

Is it possible that you changed these settings in the conf file since you generated the log file? If so, could you rerun METplus with `LOG_LEVEL=DEBUG` and send over the updated files? I'm only seeing INFO level logs.

**Reply by georgemccabe** [reply]
Based on your settings in the MET config file, I think you will want to set the following variables and remove the other ENSEMBLE_STAT_CLIMO_MEAN/STDEV_* variables:

```
ENSEMBLE_STAT_CLIMO_MEAN_FILE_NAME = {climpath}/mean_{valid?fmt=%m%d}
ENSEMBLE_STAT_CLIMO_MEAN_USE_FCST = True
ENSEMBLE_STAT_CLIMO_STDEV_FILE_NAME = {climpath}/stdev_{valid?fmt=%m%d}
ENSEMBLE_STAT_CLIMO_STDEV_USE_FCST = True
```

**Comment by BinbinZhou-NOAA** [comment]
Do you mean remove all of below?
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_METHOD = BILIN
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_WIDTH = 2
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_VLD_THRESH = 0.5
ENSEMBLE_STAT_CLIMO_MEAN_REGRID_SHAPE = SQUARE
ENSEMBLE_STAT_CLIMO_MEAN_FIELD =
ENSEMBLE_STAT_CLIMO_MEAN_TIME_INTERP_METHOD = DW_MEAN
ENSEMBLE_STAT_CLIMO_MEAN_MATCH_MONTH = FALSE
ENSEMBLE_STAT_CLIMO_MEAN_DAY_INTERVAL = 31
ENSEMBLE_STAT_CLIMO_MEAN_HOUR_INTERVAL = 6

ENSEMBLE_STAT_CLIMO_MEAN_INPUT_DIR = {climpath}
ENSEMBLE_STAT_CLIMO_STDEV_INPUT_DIR = {climpath}
ENSEMBLE_STAT_CLIMO_MEAN_INPUT_TEMPLATE = mean_{valid?fmt=%m%d}
ENSEMBLE_STAT_CLIMO_STDEV_INPUT_TEMPLATE = stdev_{valid?fmt=%m%d}

and just add
ENSEMBLE_STAT_CLIMO_MEAN_FILE_NAME = {climpath}/mean_{valid?fmt=%m%d}
ENSEMBLE_STAT_CLIMO_MEAN_USE_FCST = True
ENSEMBLE_STAT_CLIMO_STDEV_FILE_NAME = {climpath}/stdev_{valid?fmt=%m%d}
ENSEMBLE_STAT_CLIMO_STDEV_USE_FCST = True


On Mon, Aug 8, 2022 at 1:31 PM George McCabe ***@***.***>
wrote:

> Based on your settings in the MET config file, I think you will want to
> set the following variables and remove the other
> ENSEMBLE_STAT_CLIMO_MEAN/STDEV_* variables:
>
> ENSEMBLE_STAT_CLIMO_MEAN_FILE_NAME = {climpath}/mean_{valid?fmt=%m%d}
> ENSEMBLE_STAT_CLIMO_MEAN_USE_FCST = True
> ENSEMBLE_STAT_CLIMO_STDEV_FILE_NAME = {climpath}/stdev_{valid?fmt=%m%d}
> ENSEMBLE_STAT_CLIMO_STDEV_USE_FCST = True
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/1725#discussioncomment-3351487>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOQL7BOOX6R4QZERCTVYE76XANCNFSM55MA7ZQA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

IMSG at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

**Reply by georgemccabe** [reply]
Yes, I believe so.

**Reply by BinbinZhou-NOAA** [reply]
I re-ran the case by taking your suggestion. It is successful and the CRPSS score is generated !
Thanks!

**Reply by georgemccabe** [reply]
Great to hear! I will lock this discussion, but feel free to open a new topic if you have any other issues.
